after/plugin/compe.vim

```diff
@@ -0,0 +1 @@
+lua require'compe.lazy'.load_deferred()
```

Comment:
`Lazy.load_deferred` can only be called once, so the `exist(g:compe_after_loaded)` guard shouldn't be necessary. Unless someone wants to prevent that function being called, but I can't think of a single reason why anyone would want that.